### PR TITLE
SM-456 fix sentence casing

### PR DESCRIPTION
**Pull request by patrick-bitwarden**
Correct sentence case errors in various messages

## Type of change

```
- [X] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Code changes

- **file.ext:** Description of what was changed and why

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Review by Hinton — APPROVED** [review]
Confirmed only affects SM strings.

**Review by Thomas-Avery — DISMISSED** [review]
Looks good, thank you!

File: bitwarden_license/bit-web/src/app/secrets-manager/layout/navigation.component.html [hunk 1]
```diff
@@ -5,6 +5,6 @@
 <org-switcher [filter]="orgFilter"></org-switcher>
 <bit-nav-item icon="bwi-collection" text="Projects" route="projects"></bit-nav-item>
 <bit-nav-item icon="bwi-key" text="Secrets" route="secrets"></bit-nav-item>
-<bit-nav-item icon="bwi-wrench" text="Service Accounts" route="service-accounts"></bit-nav-item>
+<bit-nav-item icon="bwi-wrench" text="Service accounts" route="service-accounts"></bit-nav-item>
```
Comment (Hinton) [inline bitwarden_license/bit-web/src/app/secrets-manager/layout/navigation.component.html:8]: Whops we should localize these

```suggestion
<bit-nav-item icon="bwi-wrench" [text]="'serviceAccount' | i18n" route="service-accounts"></bit-nav-item>
```

**Review by Hinton — APPROVED** [review]
Thanks!

**Review by Hinton — APPROVED** [review]

**Pull request merged**